samples/app-complete-auth/csharp/README.md

```diff
@@ -103,7 +103,7 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
    - Select Configuration section.
    - Under configuration -> Add OAuth connection string.
    - Provide connection Name : for eg `ssoconnection`
-   - Select service provider ad `Azure Active Directory V2`
+   - Select service provider ad `Microsoft Entra ID V2`
```

Comment:
Need to Undo this change, Keep Azure Active Directory .

Reply:
fixed